queries/groovy/highlights.scm

```diff
@@ -1,64 +1,83 @@
-(unit
-  (identifier) @variable)
+[
+  "!in"
+  "!instanceof"
+  "as"
+  "assert"
+  "case"
+  "class"
+  "default"
+  "extends"
+  "in"
+  "instanceof"
+  "package"
+  "switch"
+] @keyword
```

Comment:
"in", "!in", "as" should be `@keyword.operator`.
"switch", "default", "case" should be in `@keyword.conditional`.